package.json

```diff
@@ -98,7 +98,7 @@
     },
     "overrides": {
       "graphql": "16.8.1",
-      "@envelop/core": "5.0.1",
+      "@envelop/core": "5.0.2",
```

Comment:
Hello @ardatan,
I had to update the version of this package here as well, because it overrides the version for all other packages in this repository as well. In my case for the `@graphql-yoga/plugin-apollo-inline-trace` package. It should be fixed somehow in the future, because it can cause other issues.